### PR TITLE
Podman requires the backing filesystem supports d_type

### PR DESCRIPTION
**Pull request by knowncitizen**

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/196/

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ from tripleo-quickstart here.
 # Pre-requisites
 
 - CentOS 7
```
Comment (imain) [inline README.md]: I would just say centos 7.5 here.  Maybe note that you must install as 7.4+ (IIRC), previous versions didn't set d_type when making the fs.

Comment (knowncitizen) [reply README.md]: How about both? This is a nasty one. Worth avoiding in the future.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,10 @@ from tripleo-quickstart here.
 
 # Pre-requisites
 
-- CentOS 7
+- CentOS 7.5 or greater (installed from 7.4 or newer)
+- file system that supports the following:
+    - Native Overlay Diff: "true"
```
Comment (booxter) [inline README.md]: perhaps you could explain how to check if it's true?

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/198/

File: README.md (hunk 2 above)
Comment (knowncitizen) [reply README.md]: Added troubleshooting section

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/201/

File: README.md [hunk 3]
```diff
@@ -131,3 +132,10 @@ virsh console domain_name
 ```
 
 To get to the bootstrap node. The username is `core` and the password is `notworking`
+
+### Determining if your filesystem supports d_type
+At the command line, try:
+```
+xfs_info /mount-point
```
Comment (booxter) [inline README.md]: does it work for other types of FS?

Comment (knowncitizen) [reply README.md]: I think this is the primary OS where this is a problem since default CentOS didn't enable until 7.4

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/205/

File: README.md (hunk 3 above)
Comment (booxter) [reply README.md]: Perhaps I am just clueless, but from the documentation we have it's not clear to me if I should check / call `xfs_info` on any other file system than XFS created in pre CentOS-7.5. Maybe it's well known that d_type thingy applies to XFS only, then I am sorry for the noise. But I have no idea what it is and how to make sure this step in troubleshooting doesn't apply to me if I have a different FS.

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/226/

**Comment by booxter** [comment]
@russellb you asked to squash commits into one. They are squashed.

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/242/

**Pull request merged**